rust/core-lib/src/plugins/manifest.rs

```diff
@@ -43,6 +44,15 @@ pub struct PluginDescription {
     pub languages: Vec<LanguageDefinition>,
 }
 
+fn platform_exec_path<'de, D: Deserializer<'de>>(deserializer: D) -> Result<PathBuf, D::Error> {
+    let exec_path = PathBuf::deserialize(deserializer)?;
+    if cfg!(windows) {
+        Ok(exec_path.with_extension("txt"))
```

Comment:
exe?

Reply:
🤦‍♂️ Wow, that is embarrassing. My fingers and my brain obviously did not agree!